torch/csrc/jit/runtime/symbolic_shape_registry.cpp

```diff
@@ -49,6 +49,43 @@ const std::string shape_compute_functions =
             out.append(elem)
           return out
 
+        def unary_one_unused_input(self: List[int], inp0: Any):
+          out: List[int] = []
+          for elem in self:
+            out.append(elem)
+          return out
+
+        def unary(self: List[int]):
+          out: List[int] = []
+          for elem in self:
+            out.append(elem)
+          return out
+
+        def view(self: List[int], sizes: List[int]):
+          # TODO: add assertions to check whether requested dims are valid
+          out: List[int] = []
+          for elem in sizes:
+            if elem == -1:
```

Comment:
i guess i'm fine unblocking this, but adding an assertion doesn't actually prevent the analysis from working. we would also have to add handling in the analysis so that if an exception is on the graph block we don't set the output. in general i would like to stay away from only partially handling the behavior of an operator; in the long-term i don't think it's much simpler than just adding complete support.